### ISSUE TITLE
Change itemfinder text or revert Dowsing Machine name change

### ISSUE THREAD

**Issue by BuffelSaft** [BuffelSaft/pokeemerald#354]
Does the GBA version even use dowsing rods? Name change may make no sense, need to check.

**Comment by Keyacom** [comment]
> Does the GBA version even use dowsing rods? Name change may make no sense, need to check.

Yes, it does, suggested by the item's icon. It's pretty similar to the RG artwork for the item, which makes it clear that this is just two rods.
